### PR TITLE
C# Strings.Equals() entry added for review

### LINKED ISSUE #1834: [Term Entry] C# Strings `.Equals()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `String.Equals()` method.

The entry should go in a new file in docs/content/c-sharp/concepts/strings/terms/equals/equals.md

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by alexwiehl**
### Description

A C# entry has been created for the Strings.Equals() method. This is the first "term" entry for C# Strings, so I have created a new folder labeled "terms" under c-sharp/concepts/strings. Closes #1834.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/strings/terms/equals/equals.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+Title: 'String.Equals()'
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/equals/equals.md:2]:
```suggestion
Title: '.Equals()'
```

File: content/c-sharp/concepts/strings/terms/equals/equals.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Description: 'Determines whether two String objects have the same value or not.'
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/equals/equals.md:3]:
```suggestion
Description: 'Determines whether two string objects have the same value or not.'
```

File: content/c-sharp/concepts/strings/terms/equals/equals.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Tags:
+  - 'Libraries'
+  - 'Arithmetic'
+  - 'Strings'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **String.Equals()** method determines whether two String objects have the same value or not.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/equals/equals.md:15]:
```suggestion
The **.Equals()** method determines whether two string objects have the same value or not.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@alexwiehl Just some minor edits to align this entry with our current standards. Tag me when you've finished making changes. Thanks

**Comment by caupolicandiaz** [comment]
@alexwiehl Just checking in on this entry, do you need any assistance with the edits?

**Review by caupolicandiaz — APPROVED** [review]
@alexwiehl I've gone ahead and made the changes I suggested.

**Comment by SSwiniarski** [comment]
@alexwiehl, we will be unable to merge this until you add the `Alex Wiehl` e-mail that created this entry to your GitHub account.  The CLA doesn't recognize the second e-mail.

File: content/c-sharp/concepts/strings/terms/equals/equals.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Equals()`** method determines whether two string objects have the same value or not.
+
+## Syntax
+
+```pseudo
+String.Equals(string1, string2)
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/equals/equals.md:20]: You also need to mention the syntax `string1,Equals(string2)` that you use in the example.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@alexwiehl I had one comment to add the other syntax for `.Equals()` that you use in the example. Do that and fix the email problem with the CLA and we should be ready to merge.

**Comment by caupolicandiaz** [comment]
@alexwiehl @SSwiniarski I've updated the syntax section. @SSwiniarski Any additional edits?

**Comment by SSwiniarski** [comment]
@caupolicandiaz we just need @alexwiehl to fix the e-mail problem with the CLA.

**Comment by yangc95** [comment]
Nice 👍 @alexwiehl here's the live link: https://www.codecademy.com/resources/docs/c-sharp/strings/equals

> t looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> 
> 
> Make sure to set your visibility to “everyone”.
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin
> 
> If the steps above do not work, it may be due to an email or local configuration issue.
> 
> In addition, we may reach out in the future if any adjustments to these entries are required.
>
> If you wish to write about a topic you're more comfortable with, feel free to join the discussion at [Find a Starting Point: Possible Content Discussion](https://discuss.codecademy.com/t/find-a-starting-point-possible-content-discussion/745868)

**Pull request merged**